contracts/Coordinator.sol

```diff
@@ -0,0 +1,24 @@
+pragma solidity ^0.5.0;
+
+import './DxMgnPool.sol';
+contract Coordinator {
+
+    DxMgnPool public dxMgnPool1;
+    DxMgnPool public dxMgnPool2;
+
+    constructor (
+        ERC20 _depositToken, 
```

Comment:
nit: should we call this ETH and GNO, since both are technical deposit and secondary token somewhat?